app/init.php

```diff
@@ -1622,6 +1624,76 @@ function getDevice($root): Device
     return $register->get('geodb');
 }, ['register']);
 
+App::setResource('ip', function ($request) {
+    return $request->getIp();
+}, ['request']);
+
+App::setResource('geodbClient', function () {
+    $client = new Client();
+    $client->addHeader('Authorization', 'Bearer ' . System::getEnv('_APP_GEO_SECRET'));
+    $client->setBaseUrl('http://appwrite-geo/v1');
+    return $client;
+});
+
+App::setResource('geoRecordDocker', function ($ip, $geodbClient) {
+    $record = null;
+    try {
+        $record = $geodbClient->fetch("/ips/{$ip}", Client::METHOD_GET);
+        $record = $record->json();
+    } catch (Throwable $th) {
```

Comment:
Do we have logger in geoDB HTTP server? If not, I think we should. Logs like these will get lost, so best if geoDB container directly reports to Sentry

Reply:
Yes, geoDB server has logger integrated.